pylint_odoo/checkers/no_modules.py

```diff
@@ -342,6 +355,10 @@ def visit_assign(self, node):
     def camelize(self, string):
         return re.sub(r"(?:^|_)(.)", lambda m: m.group(1).upper(), string)
 
+    def formatversion(self, string):
+        regx = r"(\d*[6-9])(\.\d*[0-9])(\.\d*[0-9])(\.\d*[0-9])(\.\d*[0-9])"
```

Comment:
`*` Can be use for 0 or more cases.
Use `+` instead of `*`for 1 or more cases.

This regex should be compatible with future version 10

=====
Maybe
=====
You can use `regx = r'\d+.\d+.\d+.\d+.\d+'`